<launch>
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find life_test)/arm_life_test/full_arm.xml'" />

  <node machine="test_host_root" pkg="pr2_etherCAT" type="pr2_etherCAT" args="-i eth0 -x robot_description"/>


  <!-- Calibration -->
  <node pkg="mechanism_bringup" type="calibrate.py"
        args="$(find life_test)/arm_life_test/full_arm_cal.xml"
        output="screen" />

  
  <include file="$(find life_test)/arm_life_test/run_random_poses_cb_forearm.launch" />

  <node pkg="life_test" type="forearm_effort_controller.py"
  	   args="r_forearm_roll_joint" />

</launch>
